assets/src/edit-story/components/inspector/document/pageAdvancement.js

```diff
@@ -79,9 +79,11 @@ function PageAdvancementPanel() {
     if (value !== newValue) {
       setDuration(newValue);
     }
-    updateStory({
-      properties: { defaultPageDuration: newValue },
-    });
+    if (defaultPageDuration !== newValue) {
```

Comment:
What if I am changing the duration back from my current value to something that happens to be the default? Then I cannot do that?

Reply:
Hmm, but `defaultPageDuration` is a changing value, just the default value is `DEFAULT_PAGE_DURATION`, no?

If the user has changed the value meanwhile to something else, e.g, `10`, then the value of `defaultPageDuration` will also be `10`. And then when the user wants to change it back to `7`, that should be possible.

LMK if I'm missing something.

Reply:
Perhaps a more precise name for `DEFAULT_PAGE_DURATION` would actually be `DEFAULT_DEFAULT_PAGE_DURATION`.

Reply:
Got it! Guess I confused it there :-)